### ISSUE TITLE
i915kms: symbol lkpi_kernel_fpu_begin undefined

### ISSUE THREAD

**Issue by vishwin** [freebsd/drm-kmod#99]
After freebsd/freebsd-src@469884cf04a9b92677c7c83e229ca6b8814f8b0a and freebsd/freebsd-src@792b602a337ddc5efaa5e5326d9433fe3da7f303, builds successfully but fails to kldload:
```
link_elf_obj: symbol lkpi_kernel_fpu_begin undefined
linker_load_file: /boot/modules/i915kms.ko - unsupported file type
```
Not sure if this is something to fix here or a regression in base.

For the pedantic, I am currently running freebsd/freebsd-src@ff01d6343f1; port was built against the same.

**Comment by shiorid** [comment]
While loading `i915kms` I receive this additional error:
```
Warning: memory type debugfsint leaked memory on destroy (2 allocations, 80 bytes leaked).
```

**Comment by vishwin** [comment]
Should be fixed by freebsd/freebsd-src@9340ebd404efad7068b5a45fd3b3d8bb21187b91. Haven't had a chance to test yet, but as soon as one of us confirms then we can close.
